docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -9,24 +9,84 @@ ifdef::notBuildingForSite[]
 include::menu.adoc[]
 endif::notBuildingForSite[]
 
+:imagesdir: ../../../../images/design/clustered
+
 == Introduction
 
 To enable Clustering with Terracotta, firstly you will have to start the http://terracotta.org[Terracotta server] configured with clustered storage.
 And then need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
 
+== Clustering concepts
+
+In this section we discuss some Terracotta clustering terms and concepts that users need to understand before creating cache managers with clustering support.
+
+Server off-heap resource::
+Server off-heap resources are storage resources defined at the server.
+`primary-server-resource` and `secondary-server-resource` are two named server off-heap resources defined in the sample
+server configuration in the next section on <<start-server, starting the Terracotta server>>.
+Clients can get their storage resources allocated from such instances.
```

Comment:
There is no resource allocated to clients - allocation is either for the entity - shared pools - or for the cache clustered tier.